README.md

```diff
@@ -2,77 +2,91 @@
 
 # pigeon • [![tests](https://github.com/exp-table/pigeon/actions/workflows/ci.yml/badge.svg?label=tests)](https://github.com/exp-table/pigeon/actions/workflows/ci.yml) ![license](https://img.shields.io/github/license/refcell/femplate?label=license) ![solidity](https://img.shields.io/badge/solidity-^0.8.17-lightgrey)
 
-A **Simple**, **Easy** tool to test cross-chain protocols.
+**Pigeon** is an open-source modular testing toolkit for cross-chain application development using Arbitray Message Bridges (AMBs).
 
-## What is Pigeon?
+- Simulates cross-chain transactions as close to mainnet.
+- Helps run cross-chain unit tests on forked mainnet.
+- Simulate the off-chain infrastructure of AMBs.
 
-Pigeon is a set of helper contracts, one per cross-chain protocol, that is designed to help you simulate as closely as possible how the cross-chain transaction would go.
+## Why Pigeon?
+Arbitrary Message Bridges (AMB) like LayerZero, Hyperlane, Connext, Wormhole, etc., operate alongside multiple off-chain actors. Hence mocking their entire infrastructure during unit testing is tricky for cross-chain application developers. 
 
-## Installation
+Thanks to Pigeon, which will simplify the life of cross-chain application developers by simulating the off-chain infrastructure of such AMBs, helping developers write unit testing across multiple forked networks seamlessly.
 
-To install with [**Foundry**](https://github.com/gakonst/foundry):
+By doing near mainnet testing, developers can quickly check sender authentication & other security assumptions associated with cross-chain application development.
 
-```sh
-forge install exp-table/pigeon
-```
+### Supported Bridges
 
-Set environment variables:
+| bridge | messaging | gas estimation |
+| --------- | :----------: | :---------: |
+| Hyperlane |      ✅      | ✅  |
+| LayerZero |      ✅      | ✅  |
+| Celer     |      ✅      |   |
+| Axelar    |      ✅      |   |
+| Stargate  |      ✅      |   |
 
-```sh
-ETH_MAINNET_RPC_URL=
-POLYGON_MAINNET_RPC_URL=
+## Getting Started
 
-# Optional
-ENABLE_ESTIMATES=
-```
+### Installation
 
-To enable gas estimation, from the [utils/scripts directory](./utils/scripts), run the following command:
+To install with [**Foundry(git)**](https://github.com/gakonst/foundry):
 
+```sh
+$ forge install exp-table/pigeon
 ```
-npm install
-npm run compile
-```
+Add `pigeon/=lib/pigeon/` to `remappings.txt`
 
-## Usage
+### Usage
```

Comment:
Not a fan of how the `Usage` section was reworked, I think the `main` branch version makes more sense because :

- no code comment -> functions called are explicit about what they do
- there is an example with `helpWithEstimates`
- the requirements for testing with gas estimates are explained